projects/1.20/assets/legendary-monsters/legendary_monsters/lang/zh_cn.json

```
@@ -0,0 +1,240 @@
+{
+  "item.legendary_monsters.dinosaur_bone" : "恐龙骨",
+  "item.legendary_monsters.ambushers_skin" : "伏击者之皮",
+  "item.legendary_monsters.sharp_battle_axe" : "锋锐战斧",
+  "item.legendary_monsters.anchor_handle" : "锚柄",
+  "item.legendary_monsters.reinforced_monstrous_anchor" : "加固的惊骇之锚",
+  "item.legendary_monsters.ancient_spike" : "远古尖刺",
+  "item.legendary_monsters.ancient_guardian_spawn_egg" : "远古守卫刷怪蛋",
+  "item.legendary_monsters.posessed_paladin_spawn_egg" : "堕落圣骑士刷怪蛋",
+  "item.legendary_monsters.monstrous_anchor" : "惊骇之锚",
+  "item.legendary_monsters.spiky_shield" : "尖刺盾牌",
+  "item.legendary_monsters.withered_ribcage" : "凋零肋骨",
+  "item.legendary_monsters.chorus_blade" : "紫颂剑",
+  "item.legendary_monsters.chorus_mask" : "紫颂面罩",
+  "item.legendary_monsters.chorus_crystal" : "紫颂水晶",
+  "item.legendary_monsters.chorus_cannon" : "紫颂加农炮",
+  "item.legendary_monsters.infected_chorus_fruit" : "受感染的紫颂果",
+  "item.legendary_monsters.chorus_bomb_projectile" : "紫颂炸弹",
+  "item.legendary_monsters.large_shulker_shell" : "大型潜影壳",
+  "item.legendary_monsters.shulker_shield" : "潜影盾",
+  "item.legendary_monsters.shulker_helmet" : "潜影头盔",
+  "item.legendary_monsters.the_great_frost" : "凛冬",
+  "item.legendary_monsters.dinosaur_bone_shield" : "恐龙骨盾",
+  "item.legendary_monsters.frozen_rune" : "霜冻符文",
+  "item.legendary_monsters.frostbitten_shield" : "霜冻之盾",
+  "item.legendary_monsters.mossy_chestplate" : "苔藓胸甲",
+  "item.legendary_monsters.lava_eaters_skin" : "熔岩吞噬者之皮",
+  "item.legendary_monsters.fiery_jaw" : "炽热之颚",
+  "item.legendary_monsters.void_sword" : "虚空之剑",
+  "item.legendary_monsters.dinosaur_bone_meal" : "恐龙骨粉",
+  "item.legendary_monsters.entity_warper" : "实体扭曲者",
+  "item.legendary_monsters.nature_crystal" : "自然水晶",
+  "item.legendary_monsters.warped_mushroom_cap_part" : "诡异蘑菇帽的一节",

```

```suggestion
  "item.legendary_monsters.warped_mushroom_cap_part" : "诡异菌帽的一节",
```
和原版诡异菌的联系？
这里part怎么解释，是cap的part还是作为part存在的cap

---

是boss诡异菌人的掉落物，由于其形象就是一个长着双腿的巨型诡异菌，所以个人认为这里应该是前者。

---

```suggestion
  "item.legendary_monsters.warped_mushroom_cap_part" : "诡异蘑菇帽碎片",
```
直觉上这么翻比较好，如果有其他能代替`碎片`的词也行